Makefile

```
@@ -425,6 +426,9 @@ $(PROTOC):
 $(JSONNET):
 	$(call fetch_go_bin_version,github.com/google/go-jsonnet/cmd/jsonnet,$(JSONNET_VERSION))
 
+$(JSONNET):
```

@daixiang0  This supposed to be `JSONNETFMT` or better `JSONNET_FMT`, but then you have to rename things here and there.
`jsonnetfmt` is not a go binary by the way so I suggest something like this.

```suggestion
$(JSONNETFMT):
	cd tmp && curl -Lso - https://github.com/google/jsonnet/archive/v${JSONNET_VERSION}.tar.gz | \
    tar xfz - -C . && \
    cd jsonnet-${JSONNET_VERSION} && \
    make && mv jsonnetfmt $(BIN_DIR) && \
    rm -rf /tmp/jsonnet-${JSONNET_VERSION}
```

Probably you'll need to define `BIN_DIR`.